.github/workflows/close-inactive-issues.yaml

```diff
@@ -0,0 +1,26 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch: # Allows for manual triggering if needed
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      # See https://github.com/actions/stale for
+      # details on the parameters for this action
+      - uses: actions/stale@v7
+        with:
+          debug-only: true # TODO: Delete this line when we are ready to implement this for our process.
+          only-issue-labels: "waiting for customer response"
+          days-before-issue-stale: 14
+          days-before-issue-close: 14
+          stale-issue-label: "stale"
+          stale-issue-message: "This issue is stale because it has been open for 30 days with no activity."
```

Comment:
when does an issue get marked as stale?

Reply:
Can we align this with what flutter-intellij does today? I don't think it currently has a concept of "stale" but that does look like a nice touch and could lead to a few more comments with repro steps over just closing.

Reply:
Right now it is set up to mark as stale after 14 days of inactivity, then if there are 14 more days of inactivity it will close the ticket

Reply:
@jacob314 it looks like they wait 7 days and then just close it with a message. 
Is that what you would like?
https://github.com/flutter/flutter-intellij/issues/4762#issuecomment-688400423

Reply:
should stale-issue-message say 14 instead of 30 now?